tests/patmat/t4661b.scala

```diff
@@ -0,0 +1,15 @@
+class C {
```

Comment:
The convention is to prefix the test file name by `i`. The tests starting with `t` are the ones that come from `scalac` IIRC